doc/src/site/sphinx/tutorials/aws_emr.rst

```diff
@@ -0,0 +1,64 @@
+Use Sparkling Water with Amazon EMR
+-----------------------------------
+
+To use Sparkling Water with an EMR cluster, you can use a premade H2O template.
+
+1. Log in to the `AWS Marketplace <https://aws.amazon.com/marketplace/>`__. 
+
+2. In the AMI & SaaS search bar, search for H2O, and select **H2O Artificial Intelligence** to open H2O in the marketplace. Review the information on this page. Note that the Delivery Methods section provides two options:
+
+   - **H2O Cluster of VM**: All the VPC, subnets, and network security groups are created for you.
+   - **H2O Cluster of VM - Bring your own VPC**: All the networks security groups, subnets, and internet gateways are created by the user.
+   
+   Click **Continue** after reviewing to open the Launch on EC2 page.
+
+  .. figure:: ../images/aws_h2oai.png
+      :alt: H2O Artificial Intelligence 
+
+3. The Launch on EC2 page provides information about launch options. On the Manual Launch tab:
+
+   - Select the offering that you prefer.
+   - Select the region to launch.
+   - Specify a Deployment option.
+
+   Click **Launch with CloudFormation Console** to begin creating your stack.
+
+  .. figure:: ../images/aws_launch_on_ec2.png
+     :alt: Launch On EC2 page
+
+4. On the Select Template page, enter https://s3-us-west-2.amazonaws.com/h2o-cloud/aws-template/TemplateEMR.json in the **Specify an Amazon S3 template URL** field.
+
+ This template requires an install script, which is available here: https://s3-us-west-2.amazonaws.com/h2o-cloud/aws-template/install_h2o_sw.sh. This script can be added using one of the following methods:
```

Comment:
This link https://s3-us-west-2.amazonaws.com/h2o-cloud/aws-template/install_h2o_sw.sh should be also Sparkling Water version specific. Right now it's hard-coding Sparkling Water 2.2 inside.

We need to create scripts like 

`https://s3-us-west-2.amazonaws.com/h2o-cloud/aws-template/install_sparkling_water_2.3.sh`,

`https://s3-us-west-2.amazonaws.com/h2o-cloud/aws-template/install_sparkling_water_2.2.sh` and

`https://s3-us-west-2.amazonaws.com/h2o-cloud/aws-template/install_sparkling_water_2.1.sh`

I will talk to Jorge about this @angela0xdata. The documentation should then again substitute the number in the link `https://s3-us-west-2.amazonaws.com/h2o-cloud/aws-template/install_sparkling_water_SUBST_SPARK_MAJOR_VERSION.sh`

This will ensure that even on AWS users following the documentation for Sparkling Water 2.3 get the AWS for Spark 2.3

Reply:
So I ported the AWS deployement scripts into codebase.

@angela0xdata you can change `https://s3-us-west-2.amazonaws.com/h2o-cloud/aws-template/install_h2o_sw.sh` to `https://s3.amazonaws.com/h2o-release/sparkling-water/rel-SUBST_SW_MAJOR_VERSION/SUBST_SW_MINOR_VERSION/templates/aws/install_sparkling_water_SUBST_SW_VERSION.sh`